python/examples/multiSchema/agents.py

```diff
@@ -0,0 +1,106 @@
+import os
+import sys
+
+examples_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
+if examples_path not in sys.path:
+    sys.path.append(examples_path)

```

Comment:
Is this just to make sure that you can import from `examples.*`?

Reply:
yup